cross/build-rootfs.sh

```diff
@@ -128,20 +128,52 @@ fi
 if [ -d "$__RootfsDir" ]; then
     if [ $__SkipUnmount == 0 ]; then
         umount $__RootfsDir/*
+        if [ $? -ne 0 ]; then
+            echo "Failed to unmount RootfsDir."
+            exit 1
+        fi
     fi
     rm -rf $__RootfsDir
 fi
 
 if [[ -n $__LinuxCodeName ]]; then
+    __InstallFailed=0
     qemu-debootstrap --arch $__UbuntuArch $__LinuxCodeName $__RootfsDir $__UbuntuRepo
+    if [ $? -ne 0 ]; then
+        __InstallFailed=1
```

Comment:
Can we, instead, jump to a label immediately instead of proceeding to the next command?